src/main/java/com/wynntils/hades/protocol/interfaces/adapters/IHadesServerAdapter.java

```diff
@@ -15,32 +15,32 @@ public interface IHadesServerAdapter extends IHadesAdapter {
      *
      * @param packet the packet itself
      */
-    void handleAuthentication(HCPacketAuthenticate packet);
+    default void handleAuthentication(HCPacketAuthenticate packet) {};
```

Comment:
All protocol methods should be implemented so don't add defaults that do nothing for them!